README.rst

```diff
@@ -57,10 +57,12 @@ Installation
 #. Install Docker:
 
    - If you are on Windows 10 Pro/Enterprise: Install `Docker <https://store.docker.com/editions/community/docker-ce-desktop-windows/>`_ 
-   
+
+   See this `tutorial  <https://docs.docker.com/docker-for-windows/install/>`_ to install Docker.
```

Comment:
i would indent this with the same indentation as the previous bullet, so it is more intuitive that this tutorial refers to the previous line